heuristic/alns.py

```diff
@@ -293,31 +359,16 @@ def initialize_random_state():
         """ Provides a seeded random state to ensure a deterministic output over different runs """
         return np.random.RandomState(seed=0)
 
-    def add_destroy_operator(self, operators):
-        for operator in operators:
-            self.add_operator(self.destroy_operators, operator)
-
-    def add_repair_operator(self, destroy_operator_id, repair_operators):
-        for new_operator in repair_operators:
-            self.repair_operators[destroy_operator_id][new_operator.func.__name__] = new_operator
-
     def add_destroy_and_repair_operators(self, operators):
-        for key in operators.keys():
-            self.add_destroy_operator([key])
-            self.add_repair_operator(key.func.__name__, operators[key])
 
-    @staticmethod
-    def add_operator(operators, new_operator):
```

Comment:
Have you testet and are sure that all operators are included and that they are run?

Reply:
Yes!